src/main/java/edu/tamu/app/service/manager/GitHubProjectService.java

```diff
@@ -0,0 +1,93 @@
+package edu.tamu.app.service.manager;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.kohsuke.github.GHIssue;
+import org.kohsuke.github.GHOrganization;
+import org.kohsuke.github.GHProject;
+import org.kohsuke.github.GHProject.ProjectStateFilter;
+import org.kohsuke.github.GHProjectCard;
+import org.kohsuke.github.GHProjectColumn;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.GHUser;
+
+import edu.tamu.app.cache.model.Card;
+import edu.tamu.app.cache.model.Member;
+import edu.tamu.app.cache.model.Sprint;
+import edu.tamu.app.model.ManagementService;
+
+public class GitHubProjectService extends AbstractGitHubService {
+
+    public GitHubProjectService(final ManagementService managementService) throws IOException {
+        super(managementService);
+    }
+
+    @Override
+    public List<Sprint> getActiveSprintsByScopeId(final String scopeId) throws Exception {
+        logger.info("Fetching active sprints for remote project with scope id " + scopeId);
+        List<Sprint> activeSprints = new ArrayList<Sprint>();
+        GHRepository repo = github.getRepositoryById(scopeId);
+        List<GHProject> projects = repo.listProjects(ProjectStateFilter.OPEN).asList();
+        for (GHProject project : projects) {
+            // Ignore projects without "Sprint" in the name
+            if (!project.getName().toUpperCase().contains(SPRINT)) {
+                continue;
+            }
+            String sprintId = String.valueOf(project.getId());
+            String name = project.getName();
+            String projectName = repo.getName();
+            List<Card> cards = getCards(project);
+            activeSprints.add(new Sprint(sprintId, name, projectName, cards));
+        }
+        return activeSprints;
+    }
+
+    @Override
+    public List<Sprint> getAdditionalActiveSprints() throws Exception {
+        GHOrganization organization = github.getOrganization(ORGANIZATION);
+        List<GHProject> projects = organization.listProjects(ProjectStateFilter.OPEN).asList();
```

Comment:
This also could be more elegant with returning result of organization list projects as list, filter by not containing sprint, and map to a sprint.